Podfile.lock

```diff
@@ -120,7 +120,7 @@ SPEC CHECKSUMS:
   Google: 98da4b1a60ed0beb80a290387a28333406a1eb90
   GoogleSignIn: d9ef55b10f0aa401a5de2747f59b725e4b9732ac
   GoogleToolboxForMac: 2501e2ad72a52eb3dfe7bd9aee7dad11b858bd20
-  GTMOAuth2: e8b6512c896235149df975c41d9a36c868ab7fba
+  GTMOAuth2: c77fe325e4acd453837e72d91e3b5f13116857b2
```

Comment:
these are all pod conflicts, so when you merge with master you might have to delete `Podfile.lock` and the entire `Pods/` file... @AAAstorga the downsides of committing pods 😢